sdk/core/core-client/review/core-client.api.md

```diff
@@ -327,9 +327,12 @@ export class ServiceClient {
 
 // @public
 export interface ServiceClientOptions extends CommonClientOptions {
+    apiVersion?: string;
```

Comment:
I would prefer to generate this instead of adding it in ServiceClientOptions, if possible.